randovania/game_description/game_patches.py

```diff
@@ -77,6 +74,9 @@ def create_from_game(cls, game: GameDescription, player_index: int, configuratio
             starting_location=game.starting_location,
             hints={},
         )
+        # todo: fix that whole function when i'm less tired
+        ret = ret.assign_dock_connections(dock_connection)
+        return ret
```

Comment:
That's also a very weird part for me. As `elevator_connection` was prefilled with `game.get_default_elevator_connection()`, I now need to prefill these elevators into `dock_connection`.
But `get_default_elevator_connection` once again needs to know what are considered elevator/teleporter for the game.
FYI: Tests are currently working but I'm still in WIP state, so please don't waste your time on reviewing the current state

Reply:
hm.... probably this idea of pre-filling the thing with unmodified elevators was a dumb.

Or we could fill dock connections with everything?
Maybe a better idea is have something that indicates what kind of connections (or weakness too) should show up in the rdvgame even when vanilla.
Maybe just none at all, the UI handles vieweing these better.

Reply:
>  Maybe a better idea is have something that indicates what kind of connections (or weakness too) should show up in the rdvgame even when vanilla. 

I let it add all elevators as before and if this should be changed it's maybe better to do it in another pr.